src/edu/csus/ecs/pc2/core/execute/Executable.java

```diff
@@ -1827,33 +1832,37 @@ protected boolean executeProgram(int dataSetNumber) {
             
             if (usingSandbox) {
```

Comment:
For robustness, useSandbox should be replacd with an accessor method like isUsingSandbox rather than a field variable.